build/checkstyle.xml

```diff
@@ -48,6 +48,9 @@
     <module name="EmptyStatement"/>
     <module name="SuperFinalize"/>
     <module name="EqualsHashCode"/>
+    <module name="EmptyCatchBlock">
+      <property name="exceptionVariableName" value="ignore|ignored"/>
```

Comment:
Maybe, but I still think it is unreasonable and it might cause conflicts with names of other variables. If we really need to have a rule I would suggest to make it less strict, e.g., it should contain "ignore" (and disregard cases).